app/components/search.tsx

```diff
@@ -68,18 +69,18 @@ export default function Search({onSelect}: Props) {
     <div>
       <input
         type="search"
-        id="searchbar"
+        className="searchbar"
         name="searchbar"
         ref={inputRef}
-        placeholder="Below are some questions to start. What other questions do you have?"
+        placeholder="Some starting questions below. Type your questions here..."
         onChange={(e) => handleChange(e.currentTarget.value, questions)}
         onFocus={() => setShowResults(true)}
         onBlur={() => setShowResults(false)} // TODO: figure out accessibility of not blurring on keyboard navigation
       />
       <div className={`dropdown ${showResults ? '' : 'hidden'}`}>
         {results.length > 0
           ? results.map(({title, score}) => <ResultItem {...{title, score, model, onSelect}} />)
-          : inputRef.current?.value && <div className="empty">(no results)</div>}
+          : inputRef.current?.value}
```

Comment:
Please remove the whole conditional expression if we don't want to display `(no results)` at all:
```diff
-        {results.length > 0
-          ? results.map(({title, score}) => <ResultItem {...{title, score, model, onSelect}} />)
-          : inputRef.current?.value && <div className="empty">(no results)</div>}
+        results.map(({title, score}) => <ResultItem {...{title, score, model, onSelect}} />)
```

Reply:
Should I remove all the references to inputRef also?

Reply:
yes please, good catch